examples/model_selection/plot_learning_curve.py

```diff
@@ -150,6 +157,7 @@ def plot_learning_curve(estimator, title, X, y, axes=None, ylim=None, cv=None,
 cv = ShuffleSplit(n_splits=100, test_size=0.2, random_state=0)
 
 estimator = GaussianNB()
+# We will use the default scoring ('accuracy'), but other could be also used.
```

Comment:
I followed the source code of [```learning_curve```](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841/sklearn/model_selection/_validation.py#L1222) to [```check_scoring```](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841d57e8b5f6b2a570312e9d832e69debc/sklearn/metrics/_scorer.py#L374).
I thinks the default scoring depends on the estimator setting, should it be specified?

Reply:
Thanks for the comments, @haochunchang !
Exactly, the default score depends on the estimator. For [classifiers the default score is mean accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.base.ClassifierMixin.html#sklearn.base.ClassifierMixin.score) (but for example for [regressors is the coefficient R^2](https://scikit-learn.org/stable/modules/generated/sklearn.base.RegressorMixin.html#sklearn.base.RegressorMixin.score)), but each estimator can then overwrite this. The [model evaluation documentation](https://scikit-learn.org/stable/modules/model_evaluation.html) has a great explanation on this.
In the case of this example in the documentation, both [GaussianNB](https://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.GaussianNB.html#sklearn.naive_bayes.GaussianNB.score) and [SVC](https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html#sklearn.svm.SVC) keep the mean accuracy. However, [`learning_curve`](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841/sklearn/model_selection/_validation.py#L1145) accepts other scorers to use instead of the default one.
I didn't add the `scoring='accuracy'` parameter when calling `plot_learning_curve` because as mentioned is the default one, but I added the comment above just to give a hint that other scorer could be used. Also the example is using [load_digits](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html#sklearn.datasets.load_digits), which creates a balanced multiclass dataset, so in this case using the `[accuracy_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html#sklearn.metrics.accuracy_score)` or the `[balanced_accuracy_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.balanced_accuracy_score.html#sklearn.metrics.balanced_accuracy_score)` would be almost the same, but for example, the `[roc_auc_score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#sklearn.metrics.roc_auc_score)` could be used.